### PR TITLE
[HE] fixing typoes all over the rulebook

### PR DESCRIPTION
**Pull request by davidpodless**
A lot of typo fixing and a little bit of phrase refinement, nothing that effect the layout of the pdf

### CODE REVIEW

File: translations/combat.tex/he.po_bkp [hunk 1]
```diff
@@ -0,0 +1,634 @@
+# Hebrew translations for Homm package
```
Comment (DarthGandalf) [inline translations/combat.tex/he.po_bkp:1]: What is this file?

**Review by DarthGandalf — CHANGES_REQUESTED** [review]

File: translations/combat.tex/he.po_bkp (hunk 1 above)
Comment (davidpodless) [reply translations/combat.tex/he.po_bkp:1]: backup I don't need anymore... deleting...

**Comment by DarthGandalf** [comment]
rm .DS_Store too

**Comment by davidpodless** [comment]
Done, next time I won't be lazy and I'll not write git add translations 😅

**Comment by qwrtln** [comment]
> Done, next time I won't be lazy and I'll not write git add translations 😅

Just add them to `.gitignore` and stay lazy.

**Pull request merged**